### PR TITLE
Add descriptions to TS

### PR DESCRIPTION
**Pull request by lukyanovas**

### CODE REVIEW

**Review by IlyaKhD — APPROVED** [review]

File: ts/framework.d.ts [hunk 1]
```diff
@@ -1,122 +1,122 @@
...
+    
+    
+    
+    
+
+    /** An object that stores information about views displayed in the application. */
     export class ViewCache {
 
-        /** @docid ViewCacheevents_viewRemoved */
+
```
Comment (San4es) [inline ts/framework.d.ts]: Where is a description for this member?
See: https://js.devexpress.com/Documentation/ApiReference/SPA_Framework/ViewCache/Events/#viewRemoved

File: ts/framework.d.ts [hunk 2]
```diff
@@ -1,122 +1,122 @@
...
-        /** @docid dxCommandOptions_renderStage */
+        /** Specifies whether the current command is rendered when a view is being rendered or after a view is shown. */
         renderStage?: string;
 
-        /** @docid dxCommandOptions_icon */
+        /** Specifies the name of the icon shown inside the widget associated with this command. */
         icon?: string;
 
-        /** @docid dxCommandOptions_iconSrc */
+
```
Comment (San4es) [inline ts/framework.d.ts]: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxAccordion/Default_Item_Template/#iconSrc

Comment (San4es) [inline ts/framework.d.ts:62]: https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/dxAccordion/Default_Item_Template/#iconSrc

File: ts/framework.d.ts (hunk 1 above)
Comment (San4es) [inline ts/framework.d.ts:30]: Where is a description for this member?
See: https://js.devexpress.com/Documentation/ApiReference/SPA_Framework/ViewCache/Events/#viewRemoved

File: ts/viz-charts.d.ts [hunk 3]
```diff
@@ -6,111 +6,111 @@ declare module DevExpress.viz.charts {
...
     export interface ChartSeries extends BaseSeries {
-        /** @docid chartSeriesObjectFields_pane */
+        /** Returns the name of the series pane. */
         pane: string;
 
-        /** @docid chartSeriesObjectFields_axis */
+        /** Returns the name of the value axis of the series. */
         axis: string;
 
         selectPoint(point: ChartPoint): void;
```
Comment (San4es) [inline ts/viz-charts.d.ts:116]: https://js.devexpress.com/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Chart_Elements/Series/Methods/#selectPointpoint

File: ts/viz-charts.d.ts [hunk 4]
```diff
@@ -119,41 +119,41 @@ declare module DevExpress.viz.charts {
         getVisiblePoints(): Array<ChartPoint>;
```
Comment (San4es) [inline ts/viz-charts.d.ts:119]: https://js.devexpress.com/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Chart_Elements/Series/Methods/#getVisiblePoints

**Review by San4es — CHANGES_REQUESTED** [review]
Some members have no description. I suggest to get them from the online documentation.

**Review by San4es — APPROVED** [review]

**Pull request merged**